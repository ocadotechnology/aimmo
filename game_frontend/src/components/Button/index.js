import styled from 'styled-components'

const Button = styled.button`
  padding: 10px
  background-color: #3886AA
  border-radius: 15px
  margin: 10px
  color: white

  &:hover {
    background-color: #2E6E8C
  }
`
export default Button
