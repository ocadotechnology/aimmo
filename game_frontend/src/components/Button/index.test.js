/* eslint-env jest */
import React from 'react'
import Button from 'components/Button'
import renderer from 'react-test-renderer'

describe('<Button />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Button>Button</Button>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
