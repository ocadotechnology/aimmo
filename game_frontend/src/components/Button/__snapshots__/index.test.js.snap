// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> renders correctly 1`] = `
<button
  className="sc-bdVaJa gmJgMj"
>
  Button
</button>
`;
